base/file.jl

```diff
@@ -890,12 +890,11 @@ function unlink(p::AbstractString)
 end
 
 # For move command
-function rename(src::AbstractString, dst::AbstractString)
+function rename(src::AbstractString, dst::AbstractString; force::Bool=false)
     err = ccall(:jl_fs_rename, Int32, (Cstring, Cstring), src, dst)
     # on error, default to cp && rm
     if err < 0
-        # force: is already done in the mv function
```

Comment:
I don't know what is meant by this comment? I think this looks like a very old vestige, when this function got split?